src/ExpressServer.js

```diff
@@ -1,17 +1,19 @@
-const express = require('express');
-const cors = require('cors');
-const boredApiRouteHandler=require('./Routes/ExperimentOne/BoredApiCaller')
+const express = require("express");
```

Comment:
looks like something went wrong with your branches... these changes shouldn't be on this branch i don't think?